azure-pipelines.yml

```diff
@@ -1,69 +0,0 @@
-trigger:
-  branches:
-    include:
-    - master
-  paths:
-    exclude:
-    - README.md
-
-jobs:
-- job: build_cross_image
-  steps:
-    - script: |
-        sudo apt-get update && sudo apt-get install -y --no-install-recommends qemu-user-static
-      displayName: Install QEMU static emulator
-    - script: |
-        docker build --build-arg PERFORMANCE_TEST_DOCKER_ARCH=arm64v8 --build-arg PERFORMANCE_TEST_TARGET_ARCH=aarch64 -t performance_test_cross -f tools/Dockerimage.crossbuild .
-      displayName: Build Docker image
-    - script: |
-        docker run -t -v `pwd`:`pwd` -w `pwd` \
-          --rm performance_test_cross \
-          sh -c "colcon build --merge-install --cmake-args -DCMAKE_TOOLCHAIN_FILE=`pwd`/tools/toolchain.cmake -DBUILD_TESTING=OFF"
-      displayName: Build performance_test
-
-- job: ubuntu_bionic
-  container:
-    image: ros:dashing
-    options: "--name ci-container -v /usr/bin/docker:/tmp/docker:ro"
-  steps:
-    - script: |
-        /tmp/docker exec -t -u 0 ci-container \
-        sh -c "apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confold" -y install sudo"
-      displayName: Set up sudo
-    - script: |
-        sudo apt-get install -y maven default-jdk
-      displayName: Install Cyclone DDS dependencies
-    - script: |
-        cd ${PIPELINE_WORKSPACE}
-        git clone https://github.com/eclipse-cyclonedds/cyclonedds.git
-        cd cyclonedds
-        git checkout 4e805597631ed0dcbdc0eecfe9d532cb75180ae7 # Head as of 20190725
-        mkdir -p build
-        cd build
-        cmake -DCMAKE_INSTALL_PREFIX=/opt/cyclonedds ../src
-        cmake --build .
-        sudo cmake --build . --target install
-        find /opt/cyclonedds
-      displayName: Build and install Cyclone DDS
```

Comment:
@esteve Can you please explain why is this removed in the gitlab CI script?

Reply:
@daggarwa I forgot about Cyclone, I'll add it to Gitlab CI

Reply:
@esteve Sounds good. Thanks!

Reply:
Fixed in 90cd76f